frontend/app/component/form/polling_station_choice/PollingStationSelector.tsx

```diff
@@ -0,0 +1,65 @@
+import { PollingStation, PollingStationsContext } from "@kiesraad/api";
+import { Icon, InputField, Spinner } from "@kiesraad/ui";
+import { useContext, useMemo, useState } from "react";
+import cls from "./PollingStationSelector.module.css";
+import { cn } from "@kiesraad/util";
+import { IconError } from "@kiesraad/icon";
+
+export function PollingStationSelector() {
+  const [pollingStationNumber, setPollingStationNumber] = useState<string>("");
+  const { pollingStations, pollingStationsLoading } = useContext(PollingStationsContext);
+
+  const currentPollingStation = useMemo(() => {
+    const parsedInt = parseInt(pollingStationNumber, 10);
+    if (pollingStationNumber !== "" && Number.isNaN(parsedInt)) {
+      return undefined;
+    }
+
+    return pollingStations.find(
+      (pollingStation: PollingStation) => pollingStation.number === parsedInt,
+    );
+  }, [pollingStationNumber, pollingStations]);
+
+  return (
+    <div className={cls.container}>
+      <InputField
+        id="pollingStation"
+        name="number"
+        value={pollingStationNumber}
+        label="Voer het nummer in:"
+        fieldWidth="narrow"
+        margin={false}
+        pattern="\d+"
```

Comment:
This line needs to be removed, otherwise it will still validate the field to only digits, but without a proper message because `title` was removed.